src/Particular.LicensingComponent/ThroughputCollector.cs

```diff
@@ -77,7 +77,7 @@ public async Task<List<EndpointThroughputSummary>> GetThroughputSummary(Cancella
             var endpointSummary = new EndpointThroughputSummary
             {
                 //want to display the endpoint name to the user if it's different to the sanitized endpoint name
-                Name = endpointGroupPerQueue.FirstOrDefault(endpoint => endpoint.Id.Name != endpoint.SanitizedName)?.Id.Name ?? endpointGroupPerQueue.Key,
+                Name = endpointGroupPerQueue.FirstOrDefault(endpoint => !string.Equals(endpoint.Id.Name, endpointGroupPerQueue.Key, StringComparison.Ordinal))?.Id.Name ?? endpointGroupPerQueue.Key,
```

Comment:
So this fixes it, but it seems `endpointGroupPerQueue.Key` could return the sanitised value?

Reply:
yes - it's meant to - that's why we are comparing the sanitized value to the endpoint name, and if the endpoint name is different, display that